nav2_system_tests/urdf/turtlebot3_burger.urdf

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" ?>
+<robot name="turtlebot3_burger" 
+  xmlns:xacro="http://ros.org/wiki/xacro">
+  <xacro:include filename="$(find turtlebot3_description)/urdf/common_properties.urdf"/>
```

Comment:
This creates a dependency on the turtlebot3_description package. I'd prefer if we don't do that.

Reply:
@mkhansen-intel Any suggestions here? Not sure how to get rid of this dependency.

Reply:
Agreed to merge so we can move forward, I believe this dependency will need to be fixed before we can merge to master so that we can run CI